frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -17,27 +17,31 @@ export function ScanErrorScreen({ error, isTestMode }: Props): JSX.Element {
   const errorMessage = (() => {
     if (!error) return undefined;
     switch (error) {
-      // Invalid interpretation
+      // Invalid ballot interpretations
       case 'invalid_test_mode':
-        return isTestMode ? 'Live Ballot detected.' : 'Test ballot detected.';
+        return isTestMode
+          ? 'Live ballot detected. Scanner is in test mode.'
+          : 'Test ballot detected. Scanner is in live mode.';
       case 'invalid_election_hash':
-        return 'Scanned ballot does not match the election this scanner is configured for.';
+        return 'The ballot does not match the election this scanner is configured for.';
       case 'invalid_precinct':
-        return 'Scanned ballot does not match the precinct this scanner is configured for.';
+        return 'The ballot does not match the precinct this scanner is configured for.';
       case 'unreadable':
-        return 'There was a problem reading this ballot. Please try again.';
       case 'unknown':
-        return undefined;
-      // Precinct scanner error
+        return 'There was a problem reading this ballot. Please scan again.';
+      // Precinct scanner errors
       case 'both_sides_have_paper':
+        return 'Scanner detected unexpected ballots in both the front and back. Remove both ballots to continue.';
       case 'paper_in_front_on_startup':
+        return 'Scanner detected an unexpected ballot in the tray. Remove ballot to continue.';
       case 'paper_in_back_on_startup':
       case 'paper_in_back_after_accept':
-        return 'Take your ballot out of the tray and try again.';
+        return 'Scanner detected an unexpected ballot at the back of the scanner. Remove ballot to continue.';
       case 'unexpected_paper_status':
+        return 'Scanner detected an unexpected ballot in scanner. Remove ballot to continue.';
```

Comment:
This case should be handled equivalently to `unexpected_event` and `plustek_error` - they have slightly different causes but all basically mean that Plustek did something weird that we didn't expect.